draft-ietf-dnsop-svcb-https.md

```diff
@@ -498,6 +502,12 @@ is the effective TargetName:
     svc2.example.net. 300   IN A     192.0.2.2
     svc2.example.net. 300   IN AAAA  2001:db8::2
 
+Some authoritative DNS implementations restrict the owner names of A and AAAA
+records.  For example, BIND 9.18 requires these owner names to be hostnames
+when the `check-names` option is enabled {{CHECK-NAMES}}.  The ServiceMode
+TargetName is required to point at A or AAAA records, so any such restrictions
+may impact the choice of TargetName, and may preclude use of the value "." if
+the ServiceMode record's owner name is not permitted for A and AAAA records.
```

Comment:
Do we want to clarify as "not permitted by the implementation for A and AAAA records"?
(It is fairly obvious already, but it seems like we want to be crystal clear that the implementation is what is doing the permitting / not permitting.)

Reply:
I think we should stick with Erik's originally proposed: 
"Some
implementations may not allow A or AAAA records on names starting with an
underscore due to various interpretations of RFCs. This could be an
operational issue when the TargetName contains an attrleaf label, as well
as using an TargetName of "." when the owner name contains an attrleaf
label."

While the additional text is useful background (and would have been good it we added it pre-WGLC/IETF LC) I think we should restrict changes to only those clearly agreed to in the thread https://mailarchive.ietf.org/arch/msg/dnsop/LhEufm0a8IjoXZ_ck0b9RTSkDTc/